packages/neo-one-website/docs/3-api-reference/2-client.md

```diff
@@ -25,18 +25,385 @@ This reference uses dot syntax to indicate static properties and `#` to indicate
 
 #### `Client`
 #### `UserAccountProvider`
+
+##### `UserAccountProvider#currentUserAccount$`
+
+An `Observable` that emits the currently selected `UserAccount`.
+
+##### `UserAccountProvider#userAccounts$`
+
+An `Observable` that emits the available `UserAccount`s.
+
+##### `UserAccountProvider#networks$`
+
+An `Observable` that emits the available networks this `UserAccountProvider` knows how to function with.
+
+##### `UserAccountProvider#getCurrentUserAccount`
+
+```typescript
+export interface UserAccountProvider {
+  readonly getCurrentUserAccount: () => UserAccount | undefined;
+}
+```
+
+@returns the currently selected `UserAccount` or `undefined` if one is not selected.
```

Comment:
Change these to just `Returns`. `@returns` has a specific meaning in jsdoc, but no meaning in markdown.